CHANGELOG.rst

```diff
@@ -1,18 +1,25 @@
-Change Log
-==========
+Changelog
+=========
+
 All notable changes to this project will be documented in this file.
-This project adheres to `Semantic Versioning <http://semver.org/>`_.
+
+The format is based on `Keep a Changelog <https://keepachangelog.com/en/1.0.0/>`_,
+and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0.html>`_.
+
+
+3.0a1_ - 2021-06-16
+-----------------
+
+- numerous changes and restructuring for target independent build
 
 
 `Unreleased`_
 -------------
 
-Added:
-
 - Started a changelog
+- Interface to the server, require 1.0 release of the server package
 
```

Comment:
Is this from the server repo?

Reply:
This line "Interface to the server, require 1.0 release of the server package" came from what was already there in master. I then added 3.0a1 as the latest version to serve as a starting point for the change log. Perhaps it was previously taken from the server repo?

Reply:
Okay, don't worry - I had not realised there was a CHANGELOG in master.